juriscraper/pacer/attachment_page.py

```diff
@@ -34,7 +33,7 @@ def query(self, document_number):
         # the attachment page.
         document_number = document_number[:3] + "0" + document_number[4:]
         url = self.url + document_number
-        logger.info(u'Querying the attachment page endpoint at URL: %s' % url)
+        logger.info(u'Querying the attachment page endpoint at URL: %s', url)
```

Comment:
Everything looks fine in this PR, so pulling, but why change this line?

Reply:
Because pylint said so!
That's why it was in the commit message!

`W: 37, 8: Specify string format arguments as logging function parameters (logging-not-lazy)`

Reply:
Woah. Neat. I guess this saves like 10 CPU cycles when the log isn't actually logged. Thanks.